build.js

```diff
@@ -25,7 +25,11 @@ const stringify = (data) =>
     // Wrap the relative parser path with require.resolve
     new RegExp(`"${resolveStart}(.*?)${resolveEnd}"`, 'g'),
     (_match, replacement) => {
-      return `require.resolve("${replacement}")`;
+      return `require.resolve("${
```

Comment:
It might be nice to add a comment here explaining this logic. It's not totally clear to me why we're switching `'@typescript-eslint'` to `'@typescript-eslint/parser'`

Reply:
(I would "suggest changes" for this, but I'm not sure what those changes would be 🙂 )

Reply:
I refactored it to no longer require this change. This change was necessary since most npm packages live at `node_modules/____`, but since this package is in an npm org, it lives at `node_modules/@____/____`. This logic handles that. I refactored it to handle it in a different way.